src/main/java/com/spotify/reaper/resources/ClusterResource.java

```diff
@@ -226,7 +181,7 @@ public Response deleteCluster(@PathParam("cluster_name") String clusterName) {
     }
     Optional<Cluster> deletedCluster = context.storage.deleteCluster(clusterName);
     if (deletedCluster.isPresent()) {
-      return Response.ok().entity(new ClusterStatus(deletedCluster.get())).build();
```

Comment:
I think the right approach to this is to reintroduce a ClusterStatus class to be returned in these responses rather than just a Collection<RepairRunStatus>. It will contain the collection of runs, but also metadata. I will do that.

Anyway, the reason I didn't return anything now, is because it would just be an empty list (no runs is enforced here). Admittedly not the proper solution.